macq/generate/pddl/generator.py

```diff
@@ -401,16 +402,27 @@ def generate_plan(self, from_ipc_file: bool = False, filename: str = None):
                     "problem": open(self.pddl_prob, "r").read(),
                 }
 
+                headers = {"persistent": "true"}
+
                 def get_api_response(delays: List[int]):
                     if delays:
                         sleep(delays[0])
                         try:
-                            resp = requests.post(
-                                "http://solver.planning.domains/solve",
-                                verify=False,
-                                json=data,
-                            ).json()
-                            return [act["name"] for act in resp["result"]["plan"]]
+                            service_url = "https://solver.planning.domains:5001/package/lama-first/solve"
+                            solve_request = requests.post(service_url, json=data, headers=headers).json()
+                            celery_result = requests.get("https://solver.planning.domains:5001/" +
+                                                         solve_request['result'])
+                            print('Computing...')
```

Comment:
```suggestion
```

Printing might be noisy, given that this may be called several times.

Reply:
you are right, i mislooked that